pkg/services/accesscontrol/models.go

```diff
@@ -62,3 +62,34 @@ func (p RoleDTO) Role() Role {
 		Created:     p.Created,
 	}
 }
+
+const (
+	// Permission actions
+
+	ActionUsersRead     = "users:read"
+	ActionUsersWrite    = "users:write"
```

Comment:
Since we have users:create for creating new users, do you think users:write is clear for editing? 🤔  Should we rename it to users:update to be consistent with `users.password:update` ?

Reply:
🤔 I think `users:write` is clear for editing. I think the `users.password:update` could be changed to `users.password:write`, I think the reason why I went with update there is because passwords cannot be read, so a `users.password:read` would never make sense.

Reply:
Sounds good to me.